setup.py

```diff
@@ -10,13 +10,13 @@
 from setuptools import find_packages
 from setuptools.command.test import test as TestCommand
 
-ld = None
-try:
-    import pypandoc
-    ld = pypandoc.convert_file('README.md', 'rst', format='markdown_github')
-except BaseException:
-    pass
-
+# Read the contents of README.md and CHANGES.md files
```

Comment:
this will work on pypi?

Reply:
I have successfully tested with:

    cd path/to/cryptofeed
    rm -rf cryptofeed.egg-info dist    # this step is important to remove unit-test files
    python3 setup.py sdist bdist_wheel

Then you can edit the file `cryptofeed.egg-info/PKG-INFO`:

    less cryptofeed.egg-info/PKG-INFO